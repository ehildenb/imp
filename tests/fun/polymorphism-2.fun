// val ( closure ( .Map , x -> let ( y = x ) and .Bindings in y | .Cases , .Bindings , .Vals ) )

// testing polymorphism

let f = fun x -> let y = x in y
in (fun x -> f) 7
