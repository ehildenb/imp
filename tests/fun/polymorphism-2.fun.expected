<FUN>
  <k>
    val ( closure ( .Map , x -> let ( y = x ) and .Bindings in y | .Cases , .Bindings , .Vals ) )
  </k>
  <env>
    .Map
  </env>
  <envs>
    .List
  </envs>
</FUN>
